packages/connectivity_plus/connectivity_plus_windows/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 1.2.0
+
+- Add ethernet as connectivity result.
+
```

Comment:
Can you please add the same changelog (mention windows package) to the main package and pump the version accordingly  + you need to pump both windows package and main package 

https://github.com/alexbatalov/plus_plugins/blob/connectivity-plus-windows-ethernet/packages/connectivity_plus/connectivity_plus/pubspec.yaml